spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe ApplicationHelper, type: :helper do
+  describe 'date formatting constants' do
+    context 'FULL_DATE_FORMAT_STRING' do
+      it 'exists' do
+        expect(ApplicationHelper::FULL_DATE_FORMAT_STRING).to be_present
+      end
+
+      it 'is a String' do
+        expect(ApplicationHelper::FULL_DATE_FORMAT_STRING).to be_a_kind_of String
+      end
+    end
+
+    context 'MONTH_YEAR_DATE_FORMAT_STRING' do
+      it 'exists' do
+        expect(ApplicationHelper::MONTH_YEAR_DATE_FORMAT_STRING).to be_present
+      end
+
+      it 'is a String' do
+        expect(ApplicationHelper::MONTH_YEAR_DATE_FORMAT_STRING).to be_a_kind_of String
+      end
+    end
+  end
+
+  context ':format_full_date' do
+    it 'formats a date' do
```

Comment:
I'd like to get the team's opinion here. My feeling is that this spec mocks too much. At least one scenario where this testing is insufficient involves the constants: if they are not set properly - e.g. `'%b%Y  -%D'` instead of `'%d %B %Y'` - the test will still pass but the actual output will be a mess. Never mind a situation where Ruby's `.strftime` changes.

My preference here would for a more functional test - verifying that given an actual date, the method returns its properly formatted string representation.